hardware_enablement/dpu-hardware-offload.adoc

```diff
@@ -0,0 +1,71 @@
+
+The DPU is a complete compute system with an independent software stack, network identity, and provisioning capabilities. The DPU is fully capable of hosting its own applications using either embedded or orchestrated deployment models.
+
+The unique capabilities of the DPU are disruptive because they allow for key infrastructure functions and their associated software stacks to be completely removed from the host node’s CPU cores and to be relocated onto the DPU.
+
+OVN/OVS flow offload is only supported on {product-version} 4.10. 
+
+:FeatureName: DPU
+//include::snippets/technology-preview.adoc[leveloffset=+0]
+
+[id="installing-configuring-accelerated-infrastruture-openshift-dpu"]
+== Installing and configuring an accelerated infrastructure with OpenShift and DPUs
+
+Installing {product-title} on a DPU makes it possible to offload packet processing from the host x86 to the DPU. Offloading resource intensive computational tasks like packet processing from the server’s CPU to the DPU can free up cycles on the OpenShift worker nodes to run the applications.
+
+include::modules/install-dpu-topology.adoc[leveloffset=+1]
+
+include::modules/assisted-installer-dpu-introduction.adoc[leveloffset=+1]
+
+include::modules/dpu-rhel-installer-node-with-podman.adoc[leveloffset=+2]
+
+include::modules/dpu-install-infrastructure-control-plane-worker-nodes.adoc[leveloffset=+1]
+
+include::modules/install-ocp-dpu-generating-the-discovery-iso-with-the-assisted-installer.adoc[leveloffset=+2]
+
+include::modules/dpu-booting-machines-using-discovery-iso.adoc[leveloffset=+2]
+
+//include::modules/install-ocp-dpu-generating-the-discovery-iso-manually.adoc[leveloffset=+3]
+
+include::modules/install-ocp-dpu-installing-with-pxe.adoc[leveloffset=+2]
+
+include::modules/install-ocp-dpu-installing-with-usb-media.adoc[leveloffset=+2]
+
+include::modules/install-ocp-dpu-monitoring-the-installation-with-the-assisted-installer-infra.adoc[leveloffset=+2]
+
+include::modules/dpu-add-hosts-cluster.adoc[leveloffset=+2]
+
+include::modules/dpu-install-tenant-control-plane-worker-nodes.adoc[leveloffset=+1]
+
+include::modules/install-ocp-dpu-generating-the-discovery-iso-with-the-assisted-installer-tenant.adoc[leveloffset=+2]
+
+include::modules/dpu-booting-machines-using-discovery-iso-tenant.adoc[leveloffset=+2]
+
+include::modules/install-ocp-dpu-monitoring-the-installation-with-the-assisted-installer.adoc[leveloffset=+2] 
+
+//include::modules/install-ocp-dpu-monitoring-the-installation-manually.adoc[leveloffset=+3]
+
+include::modules/dpu-installing-the-dpu-network-operator.adoc[leveloffset=+1]
+
+//include::modules/about-sriov-dpu.adoc[leveloffset=+1]
+
+//include::modules/nw-sriov-installing-operator.adoc[leveloffset=+2]
+
+include::modules/dpu-enable-dpu-hw-offloading.adoc[leveloffset=+1]
+
+[id="additional-resources_dpu"]
+== Additional resources
```

Comment:
RH logo on images/dpu-install_arch.png is misplaced in top right boxes.

Reply:
The three connections at the Bluefield are a bit confusing. In the current dev-preview setup, we are using two.

Reply:
@bn222 We had a meeting with JessShafer about the diagram and came up with an official one.